l10n_br_base/models/res_partner_pix.py

```diff
@@ -90,18 +99,11 @@ def _normalize_phone(self, phone):
         )
         return phone
 
-    def _normalize_cnpj_cpf(self, doc_number):
+    def _normalize_cnpj_cpf(self, doc_number, partner_id):
```

Comment:
@mbcosta aproveitando também, se puder anotar o método com o `@api.model`. foi uma falha minha não ter colocado na época.

Reply:
Seriam todos os métodos _normalize? Parece funcionar sem a api.model mas pela documentação realmente deveria ter "Decorate a record-style method where self is a recordset, but its contents is not relevant, only the model is. Such a method:" https://www.odoo.com/documentation/14.0/developer/reference/addons/orm.html

Reply:
A questão de melhora acho que é só de performance mesmo, por por em todos sim, valeu!